tests/zexperimental/test_mcmc.py

```diff
@@ -913,11 +1007,6 @@ def test_HybridGibbs_updates_state_only_after_accepting_sample():
         initial_point=np.array([0.1]),
         scale=0.1,
     ),
-    cuqi.experimental.mcmc.ULA(
```

Comment:
Why is it removed from here?

Reply:
Thank you for your comment, after updating ULA to not evaluate the log density, we needed to remove the np.nan checks that @chaozg added because it is based on evaluating the log density. Hence we needed to remove the corresponding test.